src/client/clientErrorHandler.ts

```diff
@@ -28,11 +28,19 @@ export class ClientErrorHandler implements ErrorHandler {
     this.heapDumpFolder = getHeapDumpFolderFromSettings() || context.globalStorageUri.fsPath;
   }
 
-  error(_error: Error, _message: Message, _count: number): ErrorAction {
-    return ErrorAction.Continue;
+  error(error: Error, message: Message, count: number): ErrorHandlerResult {
```

Comment:
Prefix the parameter names with `_` to indicate they aren't used

Reply:
I didn't know this convention. Thanks for sharing that. fixed.